ateam_msgs/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ find_package(rosidl_default_generators REQUIRED)
 find_package(builtin_interfaces REQUIRED)
 find_package(std_msgs REQUIRED)
 find_package(geometry_msgs REQUIRED)
+find_package(sensor_msgs REQUIRED)
```

Comment:
should be fixed in latest commit. Why did this pass CI? it is because some other node package already flagged it as needed, so rosdep still installed it?

Reply:
Yeah, for a package as common as sensor_msgs, it'd be unlikely to cause a real problem in most systems.
But, if all of ROS was being built from source with this package included, for example, the dependency would be missed so colcon might try to build this package before sensor_msgs was built.